File: ARCHIVE/2018-03-28-2100/1180.py
n = input ()

mini = 100000000000
pos = 0
ans = 0
a = raw_input ()
a = a.split (' ')
#print a
for i in a:
    try:
        i = int (float(i))
    except ValueError:
        pass
    if (i < mini):
        mini = i
        ans = pos
    pos += 1
print "Menor valor: " + str(mini)
print "Posicao: " + str(ans)

File: ARCHIVE/uri/1-Beginner/1180.c
#include <stdio.h>

int i;
int size, min_pos, min_val;

int main (void){

	scanf("%d", &size);
	int array[size];

	scanf("%d", &array[0]);
	min_val = array[0];
	min_pos = 0;

	for (i = 1; i < size; i++){

		scanf("%d", &array[i]);

		if (min_val > array[i]){

			min_val = array[i];
			min_pos = i;

		}
	}

	printf("Menor valor: %d\n", min_val);
	printf("Posicao: %d\n", min_pos);

	return 0;
}
